.github/workflows/build.yml

```
@@ -84,6 +84,7 @@ jobs:
       with:
         context: ${{ matrix.service.repo }}#${{ matrix.service.sha }}
         build-args: |
+          CRYSTAL_VERSION=1.1.1
```

To make this a little more visible, it could be good to do something like:

```suggestion
          CRYSTAL_VERSION=${{ needs.platform-info.outputs.crystal-version }}
```

Then declare this up with the [platform info](https://github.com/PlaceOS/PlaceOS/pull/137/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R16-R19) either directly, or specified as a [top level `env` entry](https://docs.github.com/en/enterprise-server@2.22/actions/reference/workflow-syntax-for-github-actions#env).